samgeo/samgeo3.py

```
@@ -315,6 +317,24 @@ def set_confidence_threshold(self, threshold: float, state=None):
             )
         # For transformers backend, the threshold is stored and used during generate_masks
 
+    def _meta_autocast(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_meta_autocast`

(ANN202)